core/src/test/scala/unit/kafka/admin/ConfigCommandTest.scala

```diff
@@ -266,6 +266,16 @@ class ConfigCommandTest extends ZooKeeperTestHarness with Logging {
     new ConfigCommandOptions(optsList.toArray).checkArgs()
   }
 
+  @Test
+  def testDescribeAllBrokerConfig(): Unit = {
+    val optsList = List("--zookeeper", "localhost:9092",
```

Comment:
--zookeeper -> --bootstrap-config. You got the right port, though! Zookeeper defaults to localhost:2181.

Reply:
Oops copy-pasta error. Good catch and will fix.

Reply:
Changed to use `--bootstrap-server` instead. I have left the rest of `--zookeeper` (legacy) usage as-is for now.